modules/aggregate/lib/puppet/functions/aggregate/targets.rb

```diff
@@ -1,14 +1,14 @@
 # frozen_string_literal: true
 
 # Aggregates the key/value pairs in the results of a ResultSet into a hash
-# mapping the keys to a hash of each distinct value and the list of nodes
+# mapping the keys to a hash of each distinct value and the list of targets
 # returning that value for the key.
-Puppet::Functions.create_function(:'aggregate::nodes') do
-  dispatch :aggregate_nodes do
+Puppet::Functions.create_function(:'aggregate::targets') do
```

Comment:
I don't think it is worth this breaking change to rename the function. But I might be wrong...

Reply:
Renaming the param is also a breaking change. I'm not sure how we want to handle that

Reply:
Fwiw, our docs do say that versions of bundled modules may have breaking changes.

Reply:
We could have both available, only document `aggregate::targets`, and remove `nodes` in 2.0? just removing `aggregate::nodes` in a y release feels...unnecessarily unfriendly.